lib/validation/reply.ts

```diff
@@ -0,0 +1,12 @@
+import { reasonForValue } from "../utils/reason-for-value";
+import { ValidationError } from "./validation-error";
+
+const messageIDRegex = /^[\w-]+$/;
```

Comment:
Similar to my previous nitpick
```suggestion
const messageIDRegex = /[0-9A-F]{8}-[0-9A-F]{4}-[4][0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}/i;
```